Makefile

```diff
@@ -3,7 +3,7 @@ SHELL=/bin/bash
 test: lint install
 	# https://github.com/HumanCellAtlas/dcp-cli/issues/127
 	coverage run --source=hca -m unittest discover -v -t . -s test/upload
-	coverage run --source=hca -m unittest discover -v -t . -s test -p test_dss_*.py
```

Comment:
can you make this a separate PR and land this ASAP?  Seems like a pretty big deal.